### PR TITLE
test: adapt for server OLTP algorithm ( 'max_degree ' & kneighbor)

### PR DESCRIPTION
**Pull request by imbajin**
TODO: figure out why `kneighbor` results is not stable (done)

### CODE REVIEW

File: src/main/java/com/baidu/hugegraph/structure/traverser/CustomizedPathsRequest.java [hunk 1]
```diff
@@ -153,7 +153,7 @@ private Step() {
             this.properties = new HashMap<>();
             this.weightBy = null;
             this.defaultWeight = Traverser.DEFAULT_WEIGHT;
-            this.degree = Traverser.DEFAULT_DEGREE;
+            this.degree = Traverser.DEFAULT_MAX_DEGREE;
```
Comment (javeme) [inline src/main/java/com/baidu/hugegraph/structure/traverser/CustomizedPathsRequest.java:156]: rename to maxDegree?

File: src/main/java/com/baidu/hugegraph/structure/traverser/EdgeStep.java [hunk 2]
```diff
@@ -47,7 +47,7 @@ protected EdgeStep() {
         this.direction = Direction.BOTH;
         this.labels = new ArrayList<>();
         this.properties = new HashMap<>();
-        this.degree = Traverser.DEFAULT_DEGREE;
+        this.degree = Traverser.DEFAULT_MAX_DEGREE;
```
Comment (javeme) [inline src/main/java/com/baidu/hugegraph/structure/traverser/EdgeStep.java:50]: ditto

File: src/main/java/com/baidu/hugegraph/structure/traverser/FusiformSimilarityRequest.java [hunk 3]
```diff
@@ -61,7 +61,7 @@ private FusiformSimilarityRequest() {
         this.label = null;
         this.direction = null;
         this.minNeighbors = 0;
-        this.degree = Traverser.DEFAULT_DEGREE;
+        this.degree = Traverser.DEFAULT_MAX_DEGREE;
```
Comment (javeme) [inline src/main/java/com/baidu/hugegraph/structure/traverser/FusiformSimilarityRequest.java:64]: ditto

File: src/test/java/com/baidu/hugegraph/api/traverser/KneighborApiTest.java [hunk 4]
```diff
@@ -257,10 +259,15 @@ public void testKneighborPostWithVertex() {
...
                 ImmutableList.of(markoId, joshId),
+                ImmutableList.of(markoId, vadasId, markoId),
+                ImmutableList.of(markoId, lopId, markoId),
+                ImmutableList.of(markoId, joshId, markoId),
                 ImmutableList.of(markoId, lopId, peterId),
                 ImmutableList.of(markoId, joshId, rippleId)
         );
+        int i = 1;
         for (Path path : kneighborResult.paths()) {
+            System.out.println(path.toString() + i++);
```
Comment (javeme) [inline src/test/java/com/baidu/hugegraph/api/traverser/KneighborApiTest.java:270]: remove the print

File: src/main/java/com/baidu/hugegraph/structure/traverser/CustomizedPathsRequest.java (hunk 1 above)
Comment (imbajin) [reply src/main/java/com/baidu/hugegraph/structure/traverser/CustomizedPathsRequest.java:156]: set back this case to degree :)

File: pom.xml [hunk 5]
```diff
@@ -5,7 +5,7 @@
 
     <groupId>com.baidu.hugegraph</groupId>
     <artifactId>hugegraph-client</artifactId>
-    <version>1.9.3</version>
+    <version>1.9.4</version>
```
Comment (javeme) [inline pom.xml:8]: update travis server_commit_id in start-hugegraph-server.sh

Comment (imbajin) [reply pom.xml:8]: updated it in `.travis.yml` & CI passed

**Review by Linary — DISMISSED** [review]

**Review by javeme — APPROVED** [review]

**Review by javeme — APPROVED** [review]

**Review by zhoney — APPROVED** [review]

**Pull request merged**